docs/DeveloperGuide.md

```diff
@@ -659,86 +669,170 @@ Given below are instructions to test the app manually.
 <div markdown="block" class="alert alert-info">:information_source: **Note:**<br>
 
 These instructions only provide a starting point for testers to work on;
-testers are expected to do more *exploratory* testing.
+testers are expected to do more _exploratory_ testing.
 
 </div>
 
 ## 7.1 Launch and shutdown
 
 1. Initial launch
 
-    1. Download the `ConnectUS.jar` file and copy into an empty folder
+   1. Download the `ConnectUS.jar` file and copy into an empty folder
 
-    1. Double-click the `ConnectUS.jar` file Expected: Shows the <u>GUI</u> with a set of sample contacts. The window size may not be optimum.
+   1. Double-click the `ConnectUS.jar` file Expected: Shows the <u>GUI</u> with a set of sample contacts. The window size may not be optimum.
 
 2. Saving window preferences
 
-    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-    1. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
-
-3. _{ more test cases …​ }_
+   1. Re-launch the app by double-clicking the jar file.<br>
+      Expected: The most recent window size and location is retained.
 
 [↑ Back to top of section](#7-instructions-for-manual-testing)
 
 [↑ Back to table of contents](#table-of-contents)
 
 <div style="page-break-after: always"></div>
 
-## 7.2 Adding a person:
-1. Adding a `Person` with just name and email
-    1. Prerequisites: None
-    2. Test case: `add n/JohnDoe e/email@example.com`<br>
-       Expected: a new `Person` named JohnDoe with given email is  created. Details of the new `Person` shown in the status message. `Person` is visible in contact list.
-    3. Test case: `add n/ e/email@example.com`<br>
-       Expected: No `Person` is created. Error details shown in status message.
-    4. _{ more test cases …​ }_
+## 7.2 Adding a contact:
```

Comment:
would like your input if we should go with contact or person here, because in the test instructions itself it mentions `Person` instead of contact. i think i also defined "contacts that are added into the application" as `Person` near the architecture section

Reply:
if u defined `Person` then I think that works. I just used the same terminology from the UG. Will change to `Person`